kimchi/src/linearization.rs

```diff
@@ -128,22 +128,24 @@ pub fn constraints_expr<F: PrimeField + SquareRootField>(
 
     // lookup
     if let Some(feature_flags) = feature_flags {
+        if feature_flags.lookup_features.patterns != LookupPatterns::default() {
```

Comment:
this does not take into account runtime tables (or user-provided tables I think), so not sure I understand this check